src/Sylius/Behat/Context/Setup/SecurityContext.php

```diff
@@ -28,32 +31,60 @@
     private $securityService;
 
     /**
+     * @var Session
+     */
+    private $minkSession;
+
+    /**
+     * @var array
+     */
+    private $minkParameters;
+
+    /**
      * @var HomePage
      */
     private $homePage;
 
     /**
-     * @var Session
+     * @var TestUserFactoryInterface
      */
-    private $minkSession;
+    private $testUserFactory;
 
     /**
-     * @var array
+     * @var RepositoryInterface
```

Comment:
Are you sure, you wont need this method? I didn't check usage of this variable. :+1: